schema/src/schema.graphql

```diff
@@ -30,7 +30,7 @@ type Mutation {
   login(email: String!, password: String!): AuthPayload!
   addNewbieTask(newbieId: ID!, input: TaskInput!): Task!
   addBuddyTask(newbieId: ID!, input: TaskInput!): Task!
-  deleteTask(taskId: ID!): Task!
+  deleteTask(taskId: ID!, newbieId: ID!): Newbie!
```

Comment:
Isn't it just `TaskId` sufficient to remove a task?
Why removing a task returns a Newbie? 🤔

Reply:
Answered [here](https://github.com/BuildIt-Poland/buddy-app/pull/53#discussion_r363790338)